metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_config.py

```diff
@@ -181,6 +181,11 @@ class SnowflakeV2Config(
         description="If enabled, populates the snowflake usage statistics. Requires appropriate grants given to the role.",
     )
 
+    include_view_definitions: bool = Field(
+        default=True,
+        description="If enabled, populates the ingested views' definitions.",
+    )
```

Comment:
this should be implemented at "generation" time, not during the fetch. otherwise view lineage will break

make the change here https://github.com/datahub-project/datahub/blob/82bd3c248fd0164aa0c525a79604968010731e0f/metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_schema_gen.py#L742

Reply:
done. I used an empty string, rather than change the pdl model to optional and have it be `else None` .

Normally, I'd favor the practice of using None in this case, but I see we already have the Hive source using the empty string, so i am following suit as to make this easily backwards compatible (and a quicker change for me 😄 ).

Let me know if you'd rather have this changed to `None`.

Reply:
that's fine